src/main/java/subway/exceptions/errors/SubwayException.java

```diff
@@ -0,0 +1,20 @@
+package subway.exceptions.errors;
+
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+public class SubwayException extends RuntimeException {
+  private final SubwayErrorCode errorCode;
+
+  public SubwayException(SubwayErrorCode errorCode) {
+    this.errorCode = errorCode;
+  }
+
+  public SubwayErrorCode getErrorCode(){
```

Comment:
코드 포맷팅 확인해주시면 좋을 것 같습니다.
```suggestion
  public SubwayErrorCode getErrorCode() {
```

Reply:
저는 이전 회사에서는 klint 등을 적용해주는 내용을 sh 파일로 만들어 리뷰전에 실행시켜주는 걸 권장하는 방향으로 진행했었는데요, 선동님은 포맷팅을 IDE 내에서 통일하고 lint나 black 같은 formatter도 따로 적용해주시는 편이실까요?

Reply:
저는 현재 개발 조직에서는 IDE인 IntelliJ IDEA의 자동 포맷팅 기능만을 활용하는 편입니다.